Include/object.h

```diff
@@ -440,6 +440,7 @@ static inline void _Py_NewReference(PyObject *op)
 
 static inline void _Py_ForgetReference(PyObject *op)
 {
+    (void)op; /* may be unused, shut up -Wunused-parameter */
```

Comment:
Please use `Py_UNUSED` macro.

Reply:
I don't think it's suitable here, as the attributes may be used depending on macros, and `Py_UNUSED` is defined as follows:

```c
#ifdef __GNUC__
#define Py_UNUSED(name) _unused_ ## name __attribute__((unused))
#else
#define Py_UNUSED(name) _unused_ ## name
#endif
```
e.g.
- it changes argument name
- I'm not sure of compiler behavior when `__attribute__((unused))` argument is in fact used
- it doesn't cover other compilers (clang?) which may be subject to the same problem

Reply:
OK, you are right.